models/articles-models.js

```diff
@@ -46,3 +46,17 @@ exports.fetchArticles = () => {
       return rows;
     });
 };
+
+exports.fetchCommentsByArticleId = (id) => {
+  return db
+    .query(`SELECT * FROM comments WHERE  article_id=$1`, [id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: "Article not found!",
```

Comment:
what information does rows hold in this case? if there are 0 rows does it mean that the article doesn't exist?